Plugins/GitUIPluginInterfaces/IGitModule.cs

```diff
@@ -70,6 +74,21 @@ public interface IGitModule
         /// <summary>Indicates HEAD is not pointing to a branch (i.e. it is detached).</summary>
         bool IsDetachedHead();
 
+        /// <summary>
+        /// Convert the path for the Git executable.
+        /// For WSL Git, the path will be adjusted.
+        /// </summary>
+        /// <param name="path">Path as seen by the Git Extensions native (Windows) executable.</param>
```

Comment:
```suggestion
        /// <param name="path">The path as seen by the Git Extensions native (Windows) executable.</param>
```

Reply:
Rewrote the documentation, slightly different